packages/nx-plugin/schemas/cdk-service/files/resources/index.ts__tmpl__

```diff
@@ -0,0 +1,24 @@
+
+import { App } from 'aws-cdk-lib';
+import dotenv from 'dotenv';
+
+import {
+  defaultEnvironment,
+  region,
+  sharedParams,
+} from '@<%= workspaceName %>/serverless-configuration';
+
+import { <%= capitalize(pascalName) %>Stack } from './stack';
```

Comment:
`pascalName` should already have the first letter capitalized since it comes from `className`:
![image](https://user-images.githubusercontent.com/19605940/213724903-409b409b-2046-4059-97aa-30258af9da78.png)

Reply:
I tested it, and it works!
So you can remove the `capitalize` function and its uses

Reply:
Indeed! I removed this